practice-one/pages/index.html

```diff
@@ -589,8 +589,8 @@ <h2 class="section-title">Lets Stay In Touch</h2>
       <nav class="footer-last">
```

Comment:
What happens if I change the position of this section to the first?

Reply:
The name won't match the meaning anymore. I will fix it